ui/Screen/Settings.svelte

```diff
@@ -166,7 +166,7 @@
 </style>
 
 <SidebarLayout dataCy="page">
-  <div class="container">
+  <div class="container" data-cy="settings-page">
```

Comment:
The `SidebarLayout` element above has `dataCy="page"`. Should we change that one to `settings-page` instead, rather than name the container of the page as the page itself? Either works for me, but it would be good to know the scheme to use.